tst/providers/MeetingProvider/MeetingManager.test.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import '@testing-library/jest-dom';
+
+import {
+  DefaultDeviceController,
+  DefaultMeetingSession,
+  LogLevel,
+  MeetingSessionConfiguration,
+} from 'amazon-chime-sdk-js';
+
+import { MeetingManager } from '../../../src/providers/MeetingProvider/MeetingManager';
+import { MeetingJoinData } from '../../../src/providers/MeetingProvider/types';
+
+describe('Meeting Manager', () => {
+  let mockMeetingJoinData: MeetingJoinData;
+  let mockMeetingSessionConfiguration: MeetingSessionConfiguration;
+  let mockMeetingManager: MeetingManager;
+
+  beforeEach(() => {
+    // @ts-ignore
+    MeetingSessionConfiguration = jest.fn().mockImplementation(() => {});
+    mockMeetingSessionConfiguration = new MeetingSessionConfiguration();
+    mockMeetingJoinData = {
+      meetingSessionConfiguration: mockMeetingSessionConfiguration,
+      options: {
+        logLevel: LogLevel.WARN,
+      }
+    };
+    mockMeetingManager = new MeetingManager();
+    // @ts-ignore
+    DefaultDeviceController = jest.fn().mockReturnValue({});
+    // @ts-ignore
+    DefaultMeetingSession = jest.fn().mockReturnValue({
+      audioVideo: {
+        addObserver: jest.fn().mockReturnValue({}),
+        removeObserver: jest.fn().mockReturnValue({}),
+      },
+    });
+    mockMeetingManager.createLogger = jest.fn().mockReturnValue({});
```

Comment:
Don't think its a good practice to mock your own test class :)

Reply:
Removed all class mocks